doc/source/whatsnew/v1.0.0.rst

```diff
@@ -88,7 +88,7 @@ Categorical
 ^^^^^^^^^^^
 
 - Added test to assert the :func:`fillna` raises the correct ValueError message when the value isn't a value from categories (:issue:`13628`)
--
+- Bug in :meth:`CombinedDatetimelikeProperties.__new__` and :meth:`Properties._delegate_property_get` was resulting in properties being applied to categories rather than values (:issue:`27952`)
```

Comment:
it is not clear to a user what you are actually doing, try to use a little more obvious language, like 

using date accessors on a categorical dtyped series of datetimes was incorrectly returning a ....